docs/markdown/Creating-Linux-binaries.md

```diff
@@ -9,8 +13,8 @@ First you need to do a fresh operating system install. You can use spare hardwar
 Once you have installed the system, you need to install build dependencies for GCC. In Debian based distros this can be done with the following commands:
 
 ```console
-$ apt get build dep g++
-$ apt get install pkg config libgmp dev libmpfr dev libmpc dev
+$ apt-get build-dep g++
```

Comment:
The proper commands for the new `apt` front-end are `apt buid-dep g++`, `apt install ...`, i.e. just replacing apt-get with the new front-end.  For these particular commands nothing changes, but in the bigger scope `apt` also act as a wrapper for specific commands such as `apt-cache`, `apt-file` and friends.